src/frontend/components/property-type/datetime/edit.tsx

```diff
@@ -6,10 +6,13 @@ import { recordPropertyIsEqual } from '../record-property-is-equal.js'
 import { PropertyLabel } from '../utils/property-label/index.js'
 import allowOverride from '../../../hoc/allow-override.js'
 import { useTranslation } from '../../../hooks/index.js'
+import { PropertyType } from '../../../../backend/index.js'
+
+const formatDate = (val:any, propertyType: PropertyType) => (propertyType === 'date' ? `${val}T00:00:00` : val)
```

Comment:
`val` should probably be just a `string | null` since the data comes from the api. We probably have to check for `null` to so it doesn't convert to `nullT00:00:00`